CalibTracker/SiStripChannelGain/test/UsefullCode/TimeHistory/PlotMacro.C

```diff
@@ -413,34 +427,30 @@ void GetAverageGain(string input, string moduleName, stLayerData& layerData )
       t1->GetEntry(ientry);
       SiStripDetId SSdetId(tree_DetId);
 
-       char LayerName[255];
+      char LayerName[255];
 
       int LayerID=tree_SubDet*1000;      
-      switch(tree_SubDet){         
-         case 3:{
-            TIBDetId tibid = TIBDetId(tree_DetId);
-            LayerID += tibid.layer();
-            sprintf(LayerName,"lTIB%i",tibid.layer());
-         }break;
-         case 4:{
-            TIDDetId tidid = TIDDetId(tree_DetId);
-            LayerID += tidid.side()*100;
-            LayerID += tidid.ring();
-            sprintf(LayerName,"rTID%c%i",tidid.side()==1?'-':'+', tidid.ring());
-         }break;
-         case 5:{
-            TOBDetId tobid = TOBDetId(tree_DetId);
-            LayerID += tobid.layer();
-            sprintf(LayerName,"lTOB%i",tobid.layer());
-         }break;
-         case 6:{
-            TECDetId tecid = TECDetId(tree_DetId);
-            LayerID += tecid.side()*100;
-            LayerID += tecid.ring();
-            sprintf(LayerName,"rTEC%c%i",tecid.side()==1?'-':'+', tecid.ring());
-         }break;
+      switch (tree_SubDet) {         
+         case SiStripDetId::TIB:
+            LayerID += getTIBorTOBlayer(tree_DetId);
```

Comment:
I'm to blame for this one... I told @alesaggio to take them out because they don't matter anyway (syntax of a `switch` structure is ugly regardless, and everything until `break` will be executed, see http://en.cppreference.com/w/cpp/language/switch) - but if you guys think they are good style / clearer, feel free to put them back.